sdk/identity/identity/review/identity.api.md

```diff
@@ -93,6 +93,7 @@ export type BrowserLoginStyle = "redirect" | "popup";
 export class ChainedTokenCredential implements TokenCredential {
     constructor(...sources: TokenCredential[]);
     getToken(scopes: string | string[], options?: GetTokenOptions): Promise<AccessToken>;
+    selectedCredential: TokenCredential | undefined;
```

Comment:
I wonder if this should be

```suggestion
    selectedCredential?: TokenCredential;
```

Reply:
I will check!

Reply:
Addressed here: https://github.com/Azure/azure-sdk-for-js/pull/16683/commits/dd1776cbc76675551929bd010eebb9a5312d28db